app/dashboard/templates/increase_bounty.html

```diff
@@ -0,0 +1,109 @@
+{% comment %}
```

Comment:
Can you switch this file to use 2-space indentation?

Reply:
will do.